pootle/core/views.py

```diff
@@ -632,6 +634,9 @@ def get_context_data(self, *args, **kwargs):
         ctx, cookie_data = self.sidebar_announcements
         ctx.update(
             super(PootleBrowseView, self).get_context_data(*args, **kwargs))
+
+        language_code, project_code = split_pootle_path(self.pootle_path)[:2]
```

Comment:
wierd, i thought i responded already

I think self.language should not be a dict. Not entirely sure why it is in LanguageView, but i figure that its because a template uses `language.name` which should be `language.fullname`